recipes/atlas-schema/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "atlas-schema" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/atlas_schema-{{ version }}.tar.gz
+  sha256: 50f8fd29e958f00ff5fe8b0f2235e281952e5391e893bb38e61c2c50751b7766
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv # --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

Comment:
> * `noarch` packages can't have skips with selectors. If the selectors are necessary, please remove `noarch: python`

@kratsg this linter error is for the `# [win]` but the conda-forge team is okay with this happening for Windows. So this can be ignored.

Reply:
They will tell you it is fine. I was given this advice by the conda-forge/core team to avoid doing builds for Linux and macOS when they're equivalent.

Reply:
The issue is the first one.

> If the reason for the skip is a missing dependency, then remove this, let it fail, and the package won't be installable on Windows until that dependency is available.

Reply:
This is a really helpful summary / checklist. I might open an Issue/PR to add this to the linter later.